client/webserver/site/src/js/orderutil.ts

```diff
@@ -135,41 +137,41 @@ export function matchStatusString (m: Match) {
     // When revoked, match status is less important than pending action if still
     // active, or the outcome if inactive.
     if (m.active) {
-      if (m.redeem) return 'Revoked - Redemption Sent' // must require confirmation if active
+      if (m.redeem) return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.ID_MATCH_STATUS_REDEMPTION_SENT }) // must require confirmation if active
       // If maker and we have not redeemed, waiting to refund, assuming it's not
       // revoked while waiting for confs on an unspent/unexpired taker swap.
-      if (m.side === Maker) return 'Revoked - Refund PENDING'
+      if (m.side === Maker) return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.ID_MATCH_STATUS_REFUND_PENDING })
       // As taker, resolution depends on maker's actions while waiting to refund.
-      if (m.counterRedeem) return 'Revoked - Redeem PENDING' // this should be very brief if we see the maker's redeem
-      return 'Revoked - Refund PENDING' // may switch to redeem if maker redeems on the sly
+      if (m.counterRedeem) return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.ID_MATCH_STATUS_REDEEM_PENDING }) // this should be very brief if we see the maker's redeem
+      return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.ID_MATCH_STATUS_REFUND_PENDING }) // may switch to redeem if maker redeems on the sly
     }
     if (m.refund) {
-      return 'Revoked - Refunded'
+      return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.ID_MATCH_STATUS_REFUNDED })
     }
     if (m.redeem) {
-      return 'Revoked - Redemption Confirmed'
+      return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.ID_MATCH_STATUS_REDEMPTION_CONFIRMED })
     }
-    return 'Revoked - Complete' // i.e. we sent no swap
+    return intl.prep(intl.ID_MATCH_STATUS_REVOKED, { status: intl.ID_MATCH_STATUS_COMPLETE }) // i.e. we sent no swap
   }
 
   switch (m.status) {
     case NewlyMatched:
-      return 'Newly Matched'
+      return intl.ID_MATCH_STATUS_NEWLY_MATCHED
     case MakerSwapCast:
-      return 'Maker Swap Sent'
+      return intl.ID_MATCH_STATUS_MAKER_SWAP_CAST
     case TakerSwapCast:
-      return 'Taker Swap Sent'
+      return intl.ID_MATCH_STATUS_TAKER_SWAP_CAST
     case MakerRedeemed:
       if (m.side === Maker) {
-        return 'Redemption Sent'
+        return intl.ID_MATCH_STATUS_REDEMPTION_SENT
       }
-      return 'Maker Redeemed'
+      return intl.ID_MATCH_STATUS_MAKER_REDEEMED
     case MatchComplete:
-      return 'Redemption Sent'
+      return intl.ID_MATCH_STATUS_REDEMPTION_SENT
```

Comment:
I guess this one should be the `ID_MATCH_STATUS_MATCH_COMPLETE`

Reply:
Are these revoked? It looks like all `m.revoked` already returned.

Reply:
Maybe I don't understand what you are saying, but `Complete` looks correct here for status `MatchComplete`

Reply:
> but Complete looks correct here for status MatchComplete

Yes, we can use `Complete` was just thinking why we have been using `Redemption Sent` for `MatchComplete`.

Reply:
> I was just thinking why we have been using `Redemption Sent` for `MatchComplete`.

Because we have `MatchConfirmed` now, otherwise it's just "Redemption Sent". @martonp Will you check out this PR?

Reply:
We updated it because in the `MatchComplete` state, an order is still active (waiting for confirmation). It would be confusing to see "Match Complete" on the screen but you cannot shut the client off because an order is still active.

Reply:
I guess that makes sense.